regression/contracts/invar_check_multiple_loops/test.desc

```diff
@@ -5,8 +5,10 @@ main.c
 ^SIGNAL=0$
 ^\[main.1\] .* Check loop invariant before entry: SUCCESS$
 ^\[main.2\] .* Check that loop invariant is preserved: SUCCESS$
-^\[main.3\] .* Check loop invariant before entry: SUCCESS$
-^\[main.4\] .* Check that loop invariant is preserved: SUCCESS$
+^\[main.3\] .* Check that DECREASES clause monotonically decreases: SUCCESS$
```

Comment:
We don't capitalize the other contracts to this looks a bit odd. How about:

```
Check monotonicity of decreases clause: ...
```

Reply:
Yeah, that was exactly what I had initially. However, in that warning message, we use "decreases" twice, and I thought it might confuse the user. This is why I decided to capitalize one of the two occurrences of "decreases." But I see your point about the consistency with other contracts. I will modify the code.